java-operators/test/com/baeldung/operators/EqualityAndRealationalTest.java

```diff
@@ -0,0 +1,98 @@
+package com.baeldung.operators;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class EqualityAndRealationalTest {
+
+	
+	@Test
+	public void greaterThanorEqualToTest() {

```

Comment:
The case should be: **greaterThanOrEqualToTest**.

Reply:
done